subsys/bluetooth/controller/ll_sw/openisa/README

```diff
@@ -0,0 +1,23 @@
+BLE support on VEGAboard HW  Rev 1.0
+
+Currently the following sample applications are supported by the BLE SW LL
+and BLE host stack running on RI5CY core of RV32M1 SoC:
```

Comment:
Move the README content to a new section in `boards/riscv/rv32m1_vega/doc/index.rst`

Reply:
Right, will do this, and also add the missing limitations